src/less/components/payex/card.less

```diff
@@ -0,0 +1,37 @@
+.cards {
+    &.cards-primary {
+        .cards-icon {
+            color: @white;
+        }
+    }
+
+    &.cards-secondary {
+        &:after {
+            border-bottom: 4px solid @payex-light-blue;
+        }
+
+        > i {
+            color: @payex-light-blue;
+        }
+
+        .cards-icon {
+            color: @white;
+            background-color: @payex-light-blue;
+        }
+    }
+
+    &.cards-tertiary {
+        &:after {
+            border-bottom: 4px solid @payex-dark-blue;
+        }
+
+        > i {
+            color: @payex-dark-blue;
+        }
+
+        .cards-icon {
+            color: @white;
+            background-color: @payex-dark-blue;
```

Comment:
Should these colors be rendered out and represented on the "colors" page?

Reply:
Colors for payex is added to the identity/colors page, including the dark blue color.